### ISSUE TITLE
Install bug - Asks to install nearly daily

### ISSUE THREAD

**Issue by KevinBatdorf** [docker/for-mac#5896]
The app nags me almost daily to update. However, I always opt to install but nothing ever changes.

  - [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: I don't know where to find this?

![Screen Shot 2021-08-07 at 2 32 07 AM](https://user-images.githubusercontent.com/1478421/128590909-899b2ebd-2d9c-4e04-9b36-1704bc1665a5.png)

If you try to cancel/ "Skip this update" you are prompted to convert to a paid membership.

**Comment by nathj07** [comment]
You are not alone in this. It is really frustrating. I've seen this more than daily too.

**Comment by qerub** [comment]
See also: https://github.com/docker/for-mac/issues/5867

**Comment by pacu** [comment]
Hi! thanks for replying. I'm going to try to reproduce the same problem with the latest update notification and get back to you.

**Comment by pacu** [comment]
> Could you run the following command and attach the output, please?
> 
> `/usr/bin/log show --debug --info --style syslog --last 1d | grep [Ss]parkle`
> 
> (assuming you’ve triggered the update less than a day ago, and it failed)
> 
> You can adjust the number of days after `--last 1d` to see if there's more log

I'm having an update everyday again.  Docked does not update.

this is the output I get 
````
pacu@pacu ~ % /usr/bin/log show --debug --info --style syslog --last 1d | grep [Ss]parkle
zsh: no matches found: [Ss]parkle
````

**Comment by pacu** [comment]
/remove-lifecycle stale 

I'm still prompted to install the same update everyday and then when there's a newer update it switches to that update. I really lost track of it. But I'm no docker hardcore user so I disregard it. 

I can contribute with diagnostics if this is interesting for anyone.

**Comment by mat007** [comment]
@pacu what is the Docker Desktop version you’re running at the moment?

**Comment by pacu** [comment]
> @pacu what is the Docker Desktop version you’re running at the moment?

<img width="430" alt="image" src="https://user-images.githubusercontent.com/484008/165129339-0a046068-ed33-4932-988c-0352a93f03aa.png">

I don't know why this information can't be copied to a text form. 👎

**Comment by mat007** [comment]
So 4.1.0.

I think you should manually update, there are many bugs in the update code that have been fixed since that version, and the experience has changed as well (it’s now entirely within one screen in the Settings).
So either [download the latest version](https://www.docker.com/products/docker-desktop/) and wait to a new release (most likely next week), or if you wanted to test the update [download 4.7.0](https://docs.docker.com/desktop/mac/release-notes/#docker-desktop-470) and trigger a check for update from the menu (or the Software update section from the Settings).

> I don't know why this information can't be copied to a text form. 

This has been fixed as well since 4.1.0 😅

**Comment by pacu** [comment]
thanks @mat007 !

A bug that prevents you from upgrading and fixing it. If that's not Skynet, what is it? 😬
